charts/gitprime-app/v2.0.0/templates/dataPipeline/_deployment.tpl

```diff
@@ -0,0 +1,114 @@
+{{- /* This represents a deployment that runs the main Data Pipeline jar and moves into different modes based on */}}
+{{- /* environment variables and the like. */}}
+{{- define "dataPipeline.deploymentTemplate" }}
+{{- $globalEnvironment := .Values.dataPipeline.environment }}
+{{- $environment := .templateData.environment }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gitprime-dp-{{- template "helpers.environment.fullName" .}}-{{ .templateData.operationMode }}
+  labels:
+    app: gitprime-dp-{{- template "helpers.environment.fullName" .}}-{{ .templateData.operationMode }}
+spec:
+  replicas: {{ .templateData.replicaCount }}
+  selector:
+    matchLabels:
+      app: gitprime-dp-{{- template "helpers.environment.fullName" .}}-{{ .templateData.operationMode }}
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: gitprime-dp-{{- template "helpers.environment.fullName" .}}-{{ .templateData.operationMode }}
+    spec:
+      containers:
+        - name: gitprime-dp-{{- template "helpers.environment.fullName" .}}-{{ .templateData.operationMode }}
+          image: gp-docker.gitprime-ops.com/integrations/gitprime-data-pipeline:{{ .Values.dataPipeline.build.commitSHA }}
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: SYSTEM_ENV_PARENT
+              value: {{ quote .Values.environment.parentName }}
+            - name: SYSTEM_ENV_MODIFIER
+              value: {{ quote .Values.environment.modifierValue }}
+            - name: SYSTEM_ENV
+              value: {{ include "helpers.environment.fullName" . | quote }}
+            - name: SYSTEM_POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: SYSTEM_OPERATION_MODE
+              value: {{ quote .templateData.operationMode }}
+            - name: SYSTEM_CONFIG_SERVER_URL
+              value: {{ include "helpers.configServerURL" . }}
+            - name: GP_REQUEUE_AOD
```

Comment:
Ripple effect of removing the AOD requeue config, this env variable can be removed and we should verify that the config jsons no longer look for it.